### ISSUE TITLE
Does libass support gradient subtitles?

### ISSUE THREAD

**Issue by NearXdu** [libass/libass#314]
Hi~

I have used vsfiltermod+aegisub to make a subtitles with gradient color.

for example the ass file is : 
`{\r\1vc(b372fd,b372fd,508ee0,508ee0)\3c&H614FB2&}\N\N{\4c&H614FB2&\fs120}Hello World\N\N\N\N\N{\r\1vc(00FFFF,000000,00FFFF,000000)\c&HFFFFFF&}`

And it can be previewed as follow:

![test](https://user-images.githubusercontent.com/24826180/45027125-1aabe280-b073-11e8-9cf9-50e5f3940c84.png)

All I want to do is to burn this ass into video by using ffmepg with the followed cmd:
`./ffmpeg -ss 60 -to 120 -i test.mp4 -vf  "ass=test.ass" -y cut.mp4`

But it seems that libass cannot render tags in vsfiltermod...

I want to know that **does libass support gradient color**, I didn't find any documents...

thanks~

**Comment by astiob** [comment]
No, youʼre right: libass does not currently support VSFilterMod extensions, and it is unlikely that it ever will.

You can try to use VSFilterMod through AviSynth (which can be run through Wine if you need it on non-Windows), or you can replace the single gradient with many single-pixel lines of different colours, which is what is routinely done in soft subtitles. I think there are probably some readily available scripts for Aegisub that can do that.
